.travis.yml

```diff
@@ -3,9 +3,10 @@ sudo: false
 language: c
 os:
   - linux
-  #- osx
+  - osx
```

Comment:
I purposely excluded that, because the 128 osX nodes on Travis are mostly very backlogged. Thus, it would take hours to finish a test run :-(